src/Files.App/ViewModels/UserControls/Widgets/RecentFilesWidgetViewModel.cs

```diff
@@ -80,6 +82,8 @@ public override List<ContextMenuFlyoutItemViewModel> GetItemMenuItems(WidgetCard
 					OpacityIcon = new() { OpacityIconStyle = "ColorIconOpenWith" },
 					Tag = "OpenWithPlaceholder",
 				},
+				 new ContextMenuFlyoutItemViewModelBuilder(Commands.RunAsAdmin).Build(),
+				new ContextMenuFlyoutItemViewModelBuilder(Commands.RunAsAnotherUser).Build(),
```

Comment:
The options don't seem to be displayed 🤔

Reply:
I mean we have to create ones like in this PR
https://github.com/files-community/Files/pull/15329